docs/locale/fr/LC_MESSAGES/cryptography.po

```diff
@@ -3,157 +3,156 @@
 # This file is distributed under the same license as the Parsec package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: Parsec 1.7.2+dev\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-03-15 11:20+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"POT-Creation-Date: 2023-02-28 16:33+0100\n"
```

Comment:
I don't think this date should be changed ?

Reply:
You mean the `POT-Creation-Date` or the other header lines ? 

`POT-Creation-Date` : I just recreated the pot files (aka "po templates") for translation, so it seems logical to update the creation date of the generated pot files. When working strictly with translation (ie. po files), only the `PO-Revision-Date` will be updated.

Regarding the other headers, those are automatically updated on save by the editor (Poedit in my case). Unless they are patched somewhere else (I couldn't find that), it should be fine to updated them IMO.

Reply:
I mean `create-date`